src/electron-main.ts

```diff
@@ -193,8 +192,8 @@ async function setupGlobals(): Promise<void> {
 
     // The tray icon
     // It's important to call `path.join` so we don't end up with the packaged asar in the final path.
-    const iconFile = `element.${process.platform === 'win32' ? 'ico' : 'png'}`;
-    iconPath = path.join(resPath, "img", iconFile);
+    const iconPath = global.vectorConfig.tray_icon_path ||
```

Comment:
minor hint: split out the part after `||` to a `defaultIconPath` const for readability.

Reply:
Hope it's more clear now without the interpolation.